Application/EdFi.Ods.Api/Controllers/TokenController.cs

```diff
@@ -46,7 +100,7 @@ public async Task<IActionResult> Post([FromBody] TokenRequest tokenRequest)
                 return BadRequest(authenticationResult.TokenError);
             }
 
-            return Ok(authenticationResult.TokenResponse);
+                return Ok(authenticationResult.TokenResponse);
```

Comment:
formatting issue here, remove the spaces

Reply:
Fixed